pages/users/[id]/bids/placed.tsx

```diff
@@ -237,16 +240,18 @@ const BidPlacedPage: NextPage = () => {
                                 </Text>
                               </CancelOfferButton>
                             ) : (
-                              <Button
-                                as={Link}
-                                href={`/tokens/${item.asset.id}/bid`}
-                                variant="outline"
-                                colorScheme="gray"
-                              >
-                                <Text as="span" isTruncated>
-                                  {t('user.bid-placed.actions.new')}
-                                </Text>
-                              </Button>
+                              !item.asset.deletedAt && ( // only display new button if asset is not deleted
+                                <Button
+                                  as={Link}
+                                  href={`/tokens/${item.asset.id}/bid`}
```

Comment:
I would suggest something like that

```suggestion
                                <Button
                                  as={Link}
                                  if={!item.asset.deletedAt}
                                  href={`/tokens/${item.asset.id}/bid`}
```

Reply:
done in https://github.com/liteflow-labs/starter-kit/pull/532/commits/4207b88b2b43fc3eea62829a01054684422c895e
I use the same implementation as the admin